specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -134,6 +134,21 @@ export function Notifications({
           count: notifications.length,
         })
       : notificationsText.notificationsLoading();
+
+  const onClearAll = () => {
```

Comment:
please use `function()` syntax instead for any function that requires `{` and `}` (more than one statement)
use `const` for any function that does not need `{` and `}` (single statement)

or is this consistency too much trouble? @melton-jason what do you think?

personally, I find it helpful as by just seeing const vs function I can immediately tell whether I am dealing with a small function or not.

Ben also preferred the `function` syntax as that way it's easier to see that this is a function, rather than just regular `const` variable

Reply:
I agree that consistency is important! All throughout the frontend code base, we _should_ maintain the consistency that has been upheld.
 
Personally, I would lean more towards Ben's convention: I associate `const` variables with non-function primitives and their derivatives (Can't explain this right now, but this is probably a remnant of my familiarity with languages like Java and C#). 
Then again, if the function is an arrow function without brackets, then you will be able to infer the type quickly because it shoul'd not take up much space.

Reply:
> Personally, I would lean more towards Ben's convention: I associate const variables with non-function primitives and their derivatives (Can't explain this right now, but this is probably a remnant of my familiarity with languages like Java and C#).

I see merits in that change. Feel free to adapt any sane convention, as long as it's consistent (meaning if you want to change a convention, you better refactor all the places OR find an eslint rule that will do the refactor for you OR write a codemod to automate the refactor)